source/GlassView.Core/Models/IAllocationInfo.cs

```diff
@@ -0,0 +1,9 @@
+namespace GlassView.Core.Models;
+
+public interface IAllocationInfo
```

Comment:
I'll change all these interfaces to classes with public setters.

Reply:
Why not records? We get the added benefit of immutability.

Reply:
All our models are now immutable records :-)